support/mesh/mesh_channel/Cargo.toml

```diff
@@ -8,7 +8,7 @@ rust-version.workspace = true
 
 [dependencies]
 mesh_node.workspace = true
-mesh_protobuf.workspace = true
+mesh_protobuf = { workspace = true, features = ["std"] }
```

Comment:
Are there no other crates that depend on mesh_protobuf directly that need std?

Reply:
I think everything that needs `std` pulls in `mesh_channel`, too, almost always via `mesh`.

It wouldn't hurt to add this to a few more places, but I did run `cargo hack` and everything built fine.